pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -264,6 +271,16 @@ let
     } // optionalAttrs pulseSupport {
       use_pulseaudio = true;
       link_pulseaudio = true;
+    } // optionalAttrs useOzone {
+      use_ozone = true;
+      ozone_platform_wayland = true;
+      ozone_platform_x11 = true;
+      ozone_auto_platforms = false;
```

Comment:
I think we can drop `ozone_auto_platforms = false;` and as a result also `ozone_platform_wayland` and `ozone_platform_x11`.

This should then result in the Linux default:
```
      ozone_platform_headless = true
      ozone_platform = "x11"
      ozone_platform_wayland = true
      ozone_platform_x11 = true
      ozone_platform_gbm = true
```

[Source](https://source.chromium.org/chromium/chromium/src/+/master:ui/ozone/ozone.gni;l=67?q=ozone_platform_x11&ss=chromium%2Fchromium%2Fsrc&originalUrl=https:%2F%2Fcs.chromium.org%2F)